file/jsonops.go

```diff
@@ -103,7 +119,7 @@ func ReadRawFile(filename string) (map[string]interface{}, error) {
 	}
 	defer jFile.Close()
 
-	b, err := ioutil.ReadAll(jFile)
```

Comment:
Break this into it's own pr. Changing the underlying io operations is something that we should be ready to rollback

Reply:
That's fair but afaik there shouldn't be a need for that since with Go 1.16 ioutil.ReadAll was changed to io.ReadAll (https://github.com/golang/go/issues/40025)